src/SDKs/Batch/Management/Management.Batch.Tests/InMemoryTests/AccountTests.InMemory.cs

```diff
@@ -295,7 +295,6 @@ public void AccountUpdateThrowsExceptions()
             Assert.Throws<ValidationException>(() => client.BatchAccount.Update(null, null, new BatchAccountUpdateParameters()));
             Assert.Throws<ValidationException>(() => client.BatchAccount.Update("foo", null, new BatchAccountUpdateParameters()));
             Assert.Throws<ValidationException>(() => client.BatchAccount.Update("foo", "bar", null));
-            Assert.Throws<ValidationException>(() => client.BatchAccount.Update("invalid+", "account", new BatchAccountUpdateParameters()));
```

Comment:
Why remove this?

Reply:
Same question all the way down for this style test

Reply:
Because Mark removed the restriction of resource group.